setup/python2/Makefile.am

```diff
@@ -30,10 +30,13 @@ $(desktop_in_files): %.desktop.in: %.desktop.in.in Makefile
 	       -e "s|\@pkgdatadir\@|$(pkgdatadir)|" \
 	       $< > $@.tmp && mv $@.tmp $@
 
+desktop_files = $(desktop_in_files:.desktop.in=.desktop)
+
 desktopdir=$(datadir)/applications
-desktop_DATA = $(desktop_in_files:.desktop.in=.desktop)
+desktop_DATA = $(desktop_files)
```

Comment:
Now I'd change the desktop file name is `desktop_DATA = org.freedesktop.IBus.Engine.Anthy.Setup.desktop`